TVOAppliance/hf_upgrade.sh

```diff
@@ -17,14 +19,10 @@ function usage()
 #function to download the package and extract...
 function download_package()
 {
-	#define file id and filename which we want to download.
-	fileid=16JM1Z1jZvISwmo0Bqnj0wJSUu2C1ZJ7G
-	outfile="offline_pkgs.tar.gz"
+	echo "Downloading $OFFLINE_PKG_NAME for Yoga release"
 
-	echo "Downloading $outfile for Yoga release"
-
-	#run the wget command to download the package.
-	wget_command=`wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://docs.google.com/uc?export=download&id=$fileid' -O- | sed -rn 's/.confirm=([0-9A-Za-z_]+)./\1\n/p')&id=$fileid" -O $outfile && rm -rf /tmp/cookies.txt`
+	#run the wget command to download the package from rpm server.
+	wget_command_rpm_server=`wget --backups 0 http://trilio:XpmkpMFviqSe@repos.trilio.io:8283/triliodata-dev-qual2-4-2/offlinePkgs/$OFFLINE_PKG_NAME`
```

Comment:
we need to create a global variable for the repo directory `triliodata-dev-qual2-4-2` , so that we can set it as per our branches